README.md

```diff
@@ -56,33 +56,52 @@ xcode-select --install
 
 ### Installing FlowForge
 
-1. Create a directory to be the home of your FlowForge install. For example: `/opt/flowforge`
+1. Create a directory to be the home of your FlowForge install. For example: `/opt/flowforge` or `c:\flowforge`
 
    For Linux/MacOS:
-
     ```
     sudo mkdir /opt/flowforge
     sudo chown $USER /opt/flowforge
     ```
 
+   For Windows:
+    ```console
+    mkdir c:\flowforge
+    ```
+
 2. Download the Installer zip file from https://github.com/flowforge/installer/releases
 
 3. Unzip the downloaded file into a temporary location and copy its contents to
    the FlowForge directory
 
+   For Linux/MacOS:
    ```
     cd /tmp/
     unzip flowforge-installer-x.y.z.zip
     cp -R flowforge-install-x.y.z/* /opt/flowforge
+   ```
+
+   For Windows:
+   ```console
+    cd c:\temp
+    tar -xf flowforge-installer-x.y.z.zip
+    xcopy /E /I flowforge-install-x.y.z/* c:\flowforge
```

Comment:
The path in the zip file has been fixed to be `flowforge-installer-x.y.z` now, I merged a change for the Linux instructions this morning.

Reply:
not sure I understand Ben?

Reply:
The path inside the `flowforge-installer-x.y.z.zip` was only `flowforge-install-x.y.z` (note the missing `er` on the end of installer due to a typo when I built the zip manually. The zip is not automatically generated without the typo

Reply:
This should do it: 158555c 
🤞